### PR TITLE
making changes for install

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/2582

### CODE REVIEW

File: install/install_connected.md [hunk 1]
```diff
@@ -6,18 +6,18 @@ Red Hat Advanced Cluster Management for Kubernetes is installed using an operato
 
 You must meet the following requirements before you install Red Hat Advanced Cluster Management for Kubernetes: 
 
+* Your Red Hat OpenShift Container Platform must have access to the Red Hat Advanced Cluster Management operator in the OperatorHub catalog. 
+
 * Red Hat OpenShift Container Platform version 4.3, or later, must be deployed in your environment, and you must be logged into it with the CLI. See the [OpenShift version 4.3 documentation](https://docs.openshift.com/container-platform/4.3/welcome/index.html) or [OpenShift version 4.4 documentation](https://docs.openshift.com/container-platform/4.4/welcome/index.html).
 
-* A StorageClass is preconfigured in Red Hat OpenShift Container Platform, and is available to create storage for Red Hat Advanced Cluster Management for Kubernetes.
```
Comment (cadawson) [inline install/install_connected.md:11]: It is my understanding that the StorageClass is generally preconfigured in OCP.  You shouldn't have to always configure the StorageClass.  For that reason, can we make it a little less firm.  Something like: 
A StorageClass that is configured in Red Hat OpenShift Container Platform to create storage for Red Hat OpenShift Advanced Cluster Management for Kubernetes. **Note**: If the StorageClass is not configured, you must create and configure a StorageClass in Red Hat OpenShift Container Platform. For more information, see [Defining a StorageClass](https://docs.openshift.com/container-platform/4.4/storage/dynamic-provisioning.html#defining-storage-classes_dynamic-provisioning) in the OpenShift Container Platform documentation.
That updated link might not be the best place, but if you keep the old one, please upgrade it to 4.4. It exists, and you only have to change the number in the URL.  This is mainly an issue with bare metal, and we are not supporting bare metal on 4.3.  Thanks!

File: install/install_connected.md [hunk 2]
```diff
@@ -6,18 +6,18 @@ Red Hat Advanced Cluster Management for Kubernetes is installed using an operato
...
+  **Note**: You must create and configure a StorageClass in Red Hat OpenShift Container Platform that is available to storage for Red Hat Advanced Cluster Management for Kubernetes. For more information, see [Installing the Local Storage Operator](https://docs.openshift.com/container-platform/4.3/storage/persistent_storage/persistent-storage-local.html#local-storage-install_persistent-storage-local) for OpenShift Container Platform. 
 
 * Your Red Hat OpenShift Container Platform command line interface (CLI) must be version 4.3, or later, and configured to run `oc` commands. See [Getting started with the CLI](https://docs.openshift.com/container-platform/4.3/cli_reference/openshift_cli/getting-started-cli.html) for information about installing and configuring the Red Hat OpenShift CLI.
 
 * Your Red Hat OpenShift Container Platform permissions must allow you to create a namespace. 
 
 * You must have an Internet connection to access the dependencies for the operator.
 
-* Your Red Hat OpenShift Container Platform must have access to the Red Hat Advanced Cluster Management operator in the OperatorHub catalog.
```
Comment (cadawson) [inline install/install_connected.md:19]: Why was this bullet removed?

Comment (dockerymick) [reply install/install_connected.md:19]: @cadawson it wasn't removed, i made it the first bullet point

Comment (cadawson) [reply install/install_connected.md:19]: I see that now. Thanks.

**Pull request merged**